packages/circuit-ui/styles/style-mixins.ts

```diff
@@ -334,17 +333,33 @@ export const hideScrollbar = (): SerializedStyles => css`
 `;
 
 /**
- * Visually communicates to the user that an input is hovered, focused,
- * or active.
+ * Visually communicates to the user that an element is hovered, focused, or
+ * active; including for the disabled, invalid and warning states.
  */
 export const inputOutline = (
   args:
     | Theme
     | {
+        /**
+         * The Theme object.
+         */
```

Comment:
I think these code comments are redundant. The option names are self-explanatory.

Reply:
I can remove them, just thought they could give some consistency to how we describe props in TS

Reply:
It's a nice idea, I just don't think it really helps users here because the props are intuitive.